x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/hooks.tsx

```diff
@@ -96,13 +100,35 @@ export type FilterItems = Array<{
 
 // TODO: add more filter names here
 export type FilterName = keyof typeof FILTER_NAMES;
-export const useActionsLogFilter = () => {
+export const useActionsLogFilter = (
+  filterName: FilterName
+): {
+  items: FilterItems;
+  setItems: React.Dispatch<React.SetStateAction<FilterItems>>;
+  hasActiveFilters: boolean;
+  numActiveFilters: number;
+  numFilters: number;
+} => {
+  const isStatusesFilter = filterName === 'statuses';
   const [items, setItems] = useState<FilterItems>(
-    RESPONSE_ACTION_COMMANDS.slice().map((filter) => ({
-      key: filter,
-      label: filter === 'unisolate' ? 'release' : filter,
-      checked: undefined,
-    }))
+    isStatusesFilter
+      ? RESPONSE_ACTION_STATUS.map((filter) => ({
+          key: filter,
+          label: (
+            <StatusBadge
+              color={
+                filter === 'successful' ? 'success' : filter === 'failed' ? 'danger' : 'warning'
+              }
+              status={filter.slice(0, 1).toUpperCase().concat(filter.slice(1))}
+            />
+          ) as unknown as string,
+          checked: undefined,
+        }))
+      : RESPONSE_ACTION_COMMANDS.map((filter) => ({
+          key: filter,
+          label: filter === 'unisolate' ? 'release' : filter,
```

Comment:
Same here. are these value visible? if so, they need to be `i18n`

Reply:
Also - for this and the command above - it might be good to create a map (object) that holds the i18n for these values so that it can be used across the different views we have.  Something like `Readonly< Record<keyof RESPONSE_ACTION_COMMANDS, string> >`

I see that you have a function in `x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_log.tsx` that has the `i18n` values, so maybe lift that into a module from where it can be reused. Maybe in a module under `x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list`? or higher in the directory chain if it is also used outside of the log list component.

Reply:
Translating the exact command doesn't make sense. We show then as is in the options list. So the filter option has `kill-process` instead of `Kill process`.

Reply:
ahh. Ok. I guess thats ok for now, although we will be displaying the API `command` not necessarily the command they entered on the Response console. I guess thats ok.

Reply:
Oh, I see what you mean. Yeah, we're showing only the API `command` in the filter options.